src/deprecation.js

```diff
@@ -918,6 +918,24 @@ Object.defineProperty(core.utils, '_saidHello', {
 });
 
 /**
+ * Adds hooks for finding and uploading items.
+ *
+ * @param {Function} [addHook] - Function call that takes two parameters: `item:*, queue:Array`
```

Comment:
Doesn't seem to like that

```
  920:1  warning  Missing JSDoc for parameter 'addHook'     valid-jsdoc
  920:1  warning  Missing JSDoc for parameter 'uploadHook'  valid-jsdoc
```

I checked and other methods such as `PIXI.Texture.setFrame(frame)`  have the params .

Its quite possible that I implemented the deprecation method in a less than ideal way. Took me a few mins to work out whats going on in there